src/docs/cookbook-manage/implement-delete-user/index.md

```diff
@@ -0,0 +1,574 @@
+ingress:
+  This article guides you in implementing a feature where a user can delete
+  their own account through the FTW template Account settings section.
+published: true
+---
+
+Starting from 2022-03, Sharetribe Flex Marketplace API has an endpoint
+for
+[deleting a user's account](https://www.sharetribe.com/api-reference/marketplace.html#delete-user).
+After the user has been deleted, any transactions where they are a
+participant cannot transition forward, and no payouts will be made to
+the user – even if the payouts have already been scheduled.
+
+This cookbook guides you in implementing an account deleting feature in
+the FTW template "Account settings" section. When the signed-in user
+navigates to the account deletion page, they need to enter their
+password to confirm that they do in fact want to delete their account.
+An endpoint in the FTW application's server then gets called, and that
+endpoint checks whether the user has incomplete transactions.
+
+Depending on the result the endpoint then either returns the count of
+incomplete transactions, or deletes the user's account through the
+`current_user/delete` endpoint. If the user cannot be deleted, the page
+shows the count of unfinished transactions, and if the user is deleted,
+they are immediately logged out.
+
+This cookbook is based on the FTW-daily template and
+`flex-default-process`. As you implement the cookbook, you will need to
+review the transaction processes used on your marketplace to see which
+transitions count as non-final transitions, i.e. ones where you do not
+want to allow the user to delete their profile.
+
+![Non-final transitions in flex-default-process](non-final-transitions.png)
+
+In addition to incomplete transactions, your marketplace transaction
+processes may have payout considerations. If your transaction process
+uses the default Stripe integration including payouts, it may take up to
+7 days from `stripe-confirm-payment-intent` action until the payout is
+processed by Stripe, even if `stripe-create-payout` is called and the
+payout has been scheduled.
+
+In other words, if your transaction processes create a provider payout
+close to the payment intent being confirmed, you may want to prevent
+users from deleting their profile before 7 days have passed from payment
+intent being confirmed. Alternatively, you can of course handle payouts
+manually for deleted users, if the transactions have otherwise been
+completed.
+
+## Add UI components
+
+### Add "Delete account" tab to Account settings
+
+First, add a new tab to the Account settings section of the template.
+
+```shell
+└── src
+    └── components
+         └── LayoutWrapperAccountSettingsSideNav
+              └── LayoutWrapperAccountSettingsSideNav.js
+          ...
+```
+
+The new page will appear as the final tab in Account Settings side
+navigation.
+
+```diff
+  const tabs = [
+    {
+      text: <FormattedMessage id="LayoutWrapperAccountSettingsSideNav.contactDetailsTabTitle" />,
```

Comment:
This translation doesn't get added at any point

Reply:
Good catch, fixed!